node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeRepoStats.java

```diff
@@ -0,0 +1,73 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.provision;
+
+import com.yahoo.config.provision.NodeResources;
+import com.yahoo.vespa.hosted.provision.autoscale.Load;
+import com.yahoo.vespa.hosted.provision.autoscale.NodeMetricSnapshot;
+
+import java.time.Duration;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Stats about the current state known to this node repo
+ *
+ * @author bratseth
+ */
+public class NodeRepoStats {
+
+    private final Load load;
+    private final Load activeLoad;
+
+    private NodeRepoStats(Load load, Load activeLoad) {
+        this.load = load;
+        this.activeLoad = activeLoad;
+    }
+
+    /**
+     * Returns the current average work-extracting utilization in this node repo over all nodes.
+     * Capacity not allocated to active nodes are taken to have 0 utilization as it provides no useful work.
+     */
+    public Load load() { return load; }
+
+    /** Returns the current average utilization in this node repo over all active nodes. */
+    public Load activeLoad() { return activeLoad; }
+
+    public static NodeRepoStats computeOver(NodeRepository nodeRepository) {
+        NodeList allNodes = nodeRepository.nodes().list();
+
+        NodeResources totalActiveResources = new NodeResources(0, 0, 0, 0);
+        double cpu = 0, memory = 0, disk = 0;
+        for (var nodeTimeseries : nodeRepository.metricsDb().getNodeTimeseries(Duration.ofHours(1), Set.of())) {
+            Optional<Node> node = allNodes.node(nodeTimeseries.hostname());
+            if (node.isEmpty() || node.get().state() != Node.State.active) continue;
+
+            Optional<NodeMetricSnapshot> snapshot = nodeTimeseries.last();
+            if (snapshot.isEmpty()) continue;
+
+            cpu += snapshot.get().cpu() * node.get().resources().vcpu();
```

Comment:
That would be a measure of "waste", not "utilization". That could be useful too. I didn't really consider it, but now I'm going to.
Utilization is also useful though, (more interpretable, and it could be that the 0.4 number is wrong), so I suggest this can be merged.